packages/bitcore-node/src/modules/ethereum/models/block.ts

```diff
@@ -178,6 +181,50 @@ export class EthBlockModel extends BaseBlock<IEthBlock> {
     }
     return JSON.stringify(transform);
   }
+
+  async verifySyncHeight(params: { chain: string; network: string; startHeight?: number }): Promise<number[]> {
```

Comment:
Seems like this could be more accurately named "getUnsyncedBlocks" or something like that. I see later you call it "gaps". Maybe "getSyncedBlockGaps".